### PR TITLE
planning: adjust max_valid_stop_distance conf for stop-sign scenario

### PR DESCRIPTION
**Pull request by jmtao**

### CODE REVIEW

File: modules/planning/scenarios/BUILD [hunk 1]
```diff
@@ -24,9 +24,11 @@ cc_library(
     name = "stage",
     srcs = [
         "stage.cc",
+        "stage_intersection_cruise.cc",
     ],
     hdrs = [
         "stage.h",
+        "stage_intersection_cruise.h",
```
Comment (xiaoxq) [inline modules/planning/scenarios/BUILD]: Split to its own target?

File: modules/planning/scenarios/stage_intersection_cruise.h [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        right_of_way_status_(right_of_way_status) {}
+
+ private:
+  Stage::StageStatus Process(const common::TrajectoryPoint& planning_init_point,
+                             Frame* frame) override;
+
+ private:
+  Stage::StageStatus FinishStage();
+
+ private:
```
Comment (xiaoxq) [inline modules/planning/scenarios/stage_intersection_cruise.h]: Leaving one `private:` is OK.

File: modules/planning/scenarios/BUILD (hunk 1 above)
Comment (jmtao) [reply modules/planning/scenarios/BUILD]: sure. will fix it in another commit.

File: modules/planning/scenarios/stage_intersection_cruise.h (hunk 2 above)
Comment (jmtao) [reply modules/planning/scenarios/stage_intersection_cruise.h]: will be removed in my next commit

**Pull request merged**